_docs/reference/istioctl-autogen/istioctl.md

```diff
@@ -0,0 +1,37 @@
+---
+category: Reference
+title: Istioctl
+bodyclass: docs
+layout: docs
+type: markdown
+---
+## istioctl
+
+Istio control interface
+
+### Synopsis
+
+
+Istio configuration command line utility. Available configuration types: [destination-policy ingress-rule route-rule]
+
+### Options
+
+```
+  -c, --kubeconfig string                Use a Kubernetes configuration file instead of in-cluster configuration
+      --log_backtrace_at traceLocation   when logging hits line file:N, emit a stack trace (default :0)
+  -n, --namespace string                 Select a Kubernetes namespace (default "default")
+  -v, --v Level                          log level for V logs
+      --vmodule moduleSpec               comma-separated list of pattern=N settings for file-filtered logging
+```
+
+### SEE ALSO
+* [istioctl collateral](istioctl_collateral.html)	 - Generate istioctl collateral files
```

Comment:
We should mark this collateral option as a hidden option in the cobra stuff which I think will prevent it from being listed when dumping out this material. This is not something we need people to use or care about, it's just pollution within these docs IMO.